packages/launchpad/cypress/fixtures/scaffolded-files.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
We have a test in [project-setup.cy.ts](https://github.com/cypress-io/cypress/blob/691234cfdf780a3d9df3cdd43a8a04b6bf36c24e/packages/launchpad/cypress/e2e/project-setup.cy.ts#L192), we do a `verifyFiles` which we could tweak to verify the ordering as well. We could remove this fixture and have better coverage.